lib/logstash/pluginmanager/list.rb

```diff
@@ -34,5 +34,7 @@ def execute
       line += " (#{spec.version})" if verbose?
       puts(line)
     end
+
+    return 0
```

Comment:
uh?!? the return value from the execute method in clamp has not effect.... are you sure this is needed??!?